src/main/java/microsoft/exchange/webservices/data/SearchFilter.java

```diff
@@ -1480,7 +1474,7 @@ public SearchFilter getSearchFilter(int index) {
      */
     public void setSearchFilter(int index, SearchFilter searchFilter) {
       if (index < 0 || index >= this.getCount()) {
-        throw new IllegalArgumentException(Strings.IndexIsOutOfRange
+        throw new IllegalArgumentException("index is out of range."
```

Comment:
please also provide a hint which is the current index, as this would be very more helpfull I think.